cmd/kops/create_cluster.go

```diff
@@ -59,6 +59,8 @@ type CreateClusterOptions struct {
 	MasterSize           string
 	MasterCount          int32
 	NodeCount            int32
+	MasterVolumeSize     int32
```

Comment:
I thought it would be much simpler to just follow the int value, then to do any parsing on it. Aws by default assumes your entering the value as a GB, so in keeping in sync with them, we should expect it in-line with that.